.gitpod.yml

```diff
@@ -1,11 +1,12 @@
 image: gitpod/workspace-full:2022-08-04-13-40-17
 workspaceLocation: observability/observability.code-workspace
 tasks:
-  - command: |
+  - init: |
       # TODO(arthursens): We should update the workspace image so we dont need the steps below
       sudo curl -Lo /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 && sudo chmod +x /usr/bin/yq
       sudo apt update ; sudo apt-get install shellcheck apt-transport-https ca-certificates -y ; sudo update-ca-certificates
       make --always-make
+  - init: |
```

Comment:
The longest time that this takes is when you run the first pre-commit check, so I think we should add something like after the install of the hook

`$(pyenv which pre-commit) run --all-files &>/dev/null`

Reply:
nice call